### DISCUSSION TITLE
关于开启hostNetwork=true参数时，如何做到k8s集群中每个节点都能开启envoy监控主机的80端口的提问

### DISCUSSION THREAD

**Discussion by ph199999** [higress-group/higress#821] [Q&A]
关于开启hostNetwork=true参数时，如何做到k8s集群中每个节点都能开启envoy监控主机的80端口，假设集群中有两台机器，默认只有一台上面开启了envoy监控主机的80端口，现在就是如何做到两台都监控到80端口（两台上面都有envoy开启）

**Comment by johnlanni** [comment]
replica改成2，通过node反亲和打散到不同机器
